pkg/action/scan.go

```diff
@@ -482,3 +504,22 @@ func Scan(ctx context.Context, c malcontent.Config) (*malcontent.Report, error)
 	}
 	return r, nil
 }
+
+// newBar returns a configured progress bar for use in `analyze` and `scan` contexts.
+func newBar() progressbar.ProgressBar {
+	return *progressbar.NewOptions(
+		0,
+		progressbar.OptionClearOnFinish(),
```

Comment:
This option ensures that the progress bar will disappear upon finishing to preserve the original output we expect.